cmd/bee/cmd/deploy.go

```diff
@@ -14,10 +14,13 @@ import (
 
 const blocktime = 15
 
+var missingParams []string
```

Comment:
does `missingParams` needs to be global variable? could it be local to `CheckMissingFlag` func?